src/components/intern-app.html

```diff
@@ -31,24 +31,44 @@
       }
     </style>
     <header>
-      <user-list></user-list>
+      <user-list user-accounts="{{userAccounts}}"></user-list>
       <create-user></create-user>
       <edit-user></edit-user>
     </header>
   </template>
 
   <script type="module">
-    
+
     import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
-    
+    import { LocalStorage } from '../api/localStorage.js';
+
+    const LocalUsers = new LocalStorage();
+
     class InternApp extends PolymerElement {
+      constructor() {
```

Comment:
nit: move this below any of your `static get` functions, but before any other component functions.